containers/Ideas/Idea.js

```diff
@@ -114,6 +114,16 @@ export const Idea = (props) => {
       setVoteDown(0);
     }
   };
+
+  const deleteIdea = () => {
+    AsyncStorage.getItem('authToken')
+      .then((token) => API.deleteIdea(moduleId, params.pk, token))
+      .then(() => {
+        Alert.alert('Your idea was deleted.', 'Thank you for participating!',  [{ text: 'Ok' }]);
+        props.navigation.navigate('IdeaProject');
```

Comment:
@khamui But I don't use goBack(), but this. goBack didn't reload the list, but for me this did. Maybe @Rineee also needs to try and decide if it works or not. :p

Reply:
@fuzzylogic2000 oh I tried an old version locally. With the new version all is good and working fine!